### PR TITLE
AMBARI-24946 - Infra Manager: use HDFS client when writing to cloud destinations

### PR DESCRIPTION
**Pull request by kasakrisz**
## What changes were proposed in this pull request?

* Use Hdfs client for cloud storage access when archiving
* Remove direct S3 support
* Setup Hdfs to use S3 when running integration tests 

## How was this patch tested?

ITs and UTs passed

### CODE REVIEW

File: ambari-infra-manager/src/main/resources/core-site.xml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+-->
+<configuration>
+<property>
+  <name>fs.defaultFS</name>
+  <value>s3a://testbucket</value>
```
Comment (adoroszlai) [inline ambari-infra-manager/src/main/resources/core-site.xml]: Is this file needed both in `ambari-infra-manager` and `ambari-infra-manager-it`?

**Comment by asfgit** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://builds.apache.org/job/Ambari-Infra-Github-PR-Builder/27/
Test PASSed.

File: ambari-infra-manager/src/main/resources/core-site.xml (hunk 1 above)
Comment (kasakrisz) [reply ambari-infra-manager/src/main/resources/core-site.xml]: Not for production. I removed it.
